### ISSUE TITLE
Arabic "Lam-Alef" character

### ISSUE THREAD

**Issue by fujiwarat** [ibus/ibus#1646]
```
OS: Ubuntu 13.04 - Debain wheezy
Architecture: x86_64
IBus version: 1.4.2
Input method name and version: IBus 1.4.2
Python version: 2.7.4
dbus version: 1.6.8
dbus-python version:2.7.4

What steps will reproduce the problem?

 * Writing Arabic character lam-alef (ﻻ) -which represent one button of two characters,
lam (ل) and alef (ا)- after any other Arabic character that connects with the next
one.

What is the expected output?

 * It's expected to connect with the previous character, like (بلا).

What do you see instead?

 * It doesn't connect with the previous character, like (بﻻ).

Please provide any additional information below.

 * Work-Around: To fix it, write lam character then alef character, and not to use
lam-alef button.

 * same happens for lam-alef-md (ﻵ) and lam-hamza (ﻷ).

 * Suggestion: These buttons insert two distinct characters instead of one character
that is represented by the shortcut key.
```

Original issue reported on code.google.com by `Mokhtar92` on 2013-06-02 15:07:57

**Comment by fujiwarat** [comment]
Do you still have a problem?

**Comment by mike-fabian** [comment]
This is fixed in recent versions of ibus, ibus supports compose sequences with length 1 now.

**Comment by mike-fabian** [comment]
Closing this because it is fixed in recent versions of ibus (not exactly sure which version fixed this, but in ibus-1.5.28 it works).
